src/components/socialLogin/SocialLoginContainer.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { parse } from 'query-string';
+import alertify from 'alertify.js';
+import { socialLoginAction } from '../../actions/socialLogin';
+import Footer from '../Footer';
+
+/**
+ * @description handles social login
+ * @returns {*} jsx
+ *
+ */
+export class SocialLoginContainer extends Component {
+  /**
+   *
+   * @param {object} props
+   */
+  constructor(props) {
+    super(props);
+    this.state = {};
+    const { match, history } = this.props;
+    const { socialLogin } = match.params;
+    const { code } = parse(history.location.search);
+    if (!code) {
+      window.location = `http://localhost:3000/api/${socialLogin}`;
```

Comment:
`http://localhost:3000/api` should not be hard-coded but should be in an env variable